UPGRADE-7.4.md

```diff
@@ -22,6 +22,7 @@ Config
 ------
 
  * Deprecate accessing the internal scope of the loader in PHP config files, use only its public API instead
+ * Deprecate setting a default value to a node that is required, and vice versa
```

Comment:
```suggestion
 * Deprecate setting a default value to a node that is required
```

I think it's understandable like this?

Reply:
Sorry I missed your comment and merged.
I used a different wording for the 8.0 branch: https://github.com/symfony/symfony/pull/62109